### ISSUE TITLE
renderable.setMaterial() question...

### ISSUE THREAD

**Issue by Meenners** [google-ar/sceneform-android-sdk#353]
Hello everyone, quick question on setting up materials.
I am trying to modify the material at runtime after my ModelRenderable has been placed on a plane.  This is being called on a button click event, but nothing happens.  I am using a .GLB as my source type.

``` java
if (modelRenderable == null) { return; }
Material m = modelRenderable.getMaterial().makeCopy();
m.setBoolean("doubleSided",myBool);  
m.setBoolean("depthWrite",myBool);
m.setFloat("alphaCutoff ", myFloat);    // 0 -1 range
modelRenderable.setMaterial(m);
```
I did some digging and i read something about reinstaciating the renderable for my material to take change?  if thats correct, any sugesstions on the proper way to go about it?

I also read about submeshIndexs of my model, how do i go about this?  Found one single example from the community but I'm not sure how to implement it. 

Basically I just want to modify the alphaCutOff and double Sided option on the entire model regardless the number of meshes.   (the GLB should have 1 single texture and material)

Thanks!

**Comment by romainguy** [comment]
`alphaCutoff`, `doubleSided` and `depthWrite` cannot be controlled at runtime, they are built into the materials.

**Comment by Meenners** [comment]
That's what I thought.  Thank you for confirming :)  

So, if not at runtime...  is it possible to modify these values prior to creating the rederable or through re-rendering the asset?  or is that still considered runtime?

**Comment by romainguy** [comment]
You can create [custom materials](https://developers.google.com/ar/develop/java/sceneform/custom-material) to achieve this.

**Comment by romainguy** [comment]
BTW, I misspoke the alpha cutoff value can be changed dynamically, at least in the renderer. It has its own API, it's not treated as a parameter of the material. Looks like Sceneform does not expose that API though, this should get fixed.
